relay-server/src/http.rs

```diff
@@ -46,28 +46,22 @@ pub struct Request(pub reqwest::Request);
 
 pub struct RequestBuilder {
     builder: Option<reqwest::RequestBuilder>,
-    body: Option<Bytes>,
 }
 
 impl RequestBuilder {
     pub fn reqwest(builder: reqwest::RequestBuilder) -> Self {
         RequestBuilder {
             builder: Some(builder),
-            body: None,
         }
     }
 
     pub fn finish(self) -> Result<Request, HttpError> {
-        let mut builder = self.builder.unwrap();
-        if let Some(body) = self.body {
-            builder = builder.body(body);
-        }
-        Ok(Request(builder.build()?))
+        Ok(Request(self.builder.unwrap().build()?))
```

Comment:
Why is unwrapping OK here?

Reply:
The builder will never be set to `None`. The option is there to allow moving reqwest's inner builder in the builder methods.

Reply:
👍 , might be worth a code comment!